### PR TITLE
add 鉅亨財經新聞 showcase

### PR DESCRIPTION
**Pull request by blackbing**

### CODE REVIEW

**Comment by facebook-github-bot-2** [comment]
By analyzing the blame information on this pull request, we identified @brentvatne and @manonthemat to be potential reviewers.

**Comment by facebook-github-bot-2** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-8** [comment]
@blackbing updated the pull request.

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -1007,6 +1007,12 @@ var apps = [
     icon: 'http://pp.myapp.com/ma_icon/0/icon_10966186_1460087288/96',
     link: 'http://android.myapp.com/myapp/detail.htm?apkName=com.tencent.karaoke',
     author: '石玉磊',
+  },{
+    name: '鉅亨財經新聞',
+    icon: 'http://a4.mzstatic.com/us/r30/Purple18/v4/1e/b1/dd/1eb1dd6e-0d9e-03cf-22f9-1b6fd816814d/icon175x175.jpeg',
+    linkAppStore: 'https://itunes.apple.com/us/app/ju-heng-cai-jing-xin-wen/id1071014509?mt=8',
+    linkPlayStore: 'https://play.google.com/store/apps/details?id=com.cnyes.android',
+    author: '鉅亨網'
```
Comment (bestander) [inline website/src/react-native/showcase.js]: need a trailing comma here for consistency

Comment (blackbing) [inline website/src/react-native/showcase.js]: @bestander, I fixed it. thanks for your reviewing.

**Comment by facebook-github-bot-7** [comment]
@blackbing updated the pull request.

File: website/src/react-native/showcase.js [hunk 2]
```diff
@@ -1014,6 +1014,12 @@ var apps = [
     icon: 'http://pp.myapp.com/ma_icon/0/icon_10966186_1460087288/96',
     link: 'http://android.myapp.com/myapp/detail.htm?apkName=com.tencent.karaoke',
     author: '石玉磊',
+  },{
```
Comment (bestander) [inline website/src/react-native/showcase.js]: nit: put `{` on the new line please as well

**Comment by bestander** [comment]
@blackbing one more nit please.
The file is large, we need to take care of having consistent style to make it manageable.

File: website/src/react-native/showcase.js (hunk 2 above)
Comment (blackbing) [inline website/src/react-native/showcase.js]: @bestander my bad. fixed  and thanks again.

**Comment by facebook-github-bot-3** [comment]
@blackbing updated the pull request.

**Pull request closed without merging**